suites/pacific/iscsi/tier-1_cephadm_test-iscsi_multiple_deployment.yaml

```diff
@@ -0,0 +1,121 @@
+#===============================================================================================
```

Comment:
can we apply iscsi specific IOs?

Reply:
Hi Sunil,
Followed the steps mentioned in the bz, and the issue was hit when enabling and disabling iscsi multiple times. May be the test can be extended with IO's to increase the scope, but can we have that as a separate patch, as an extension to this.

Reply:
Hi Sunil,
This will be added as an enhancement. The current patch has libs required for other patches, so have requested to close this and a backlog will be created to address your comment. Thanks

Reply:
@pranavprakash20 Do create a backlog.